mantidproject/mantid
# IndirectSimulations - Post Plot and Save

The plotting and saving of the results workspace has been changed to after the execution of the algorithm for the Indirect Simulations interface.

**To test:**

Check each plot option plots a suitable graph and that the appropriate workspaces have been saved in your save directory.
Moldyn: \olympic\babylon5\Public\Louise McCann\Simulations\NaF_DISF.cdl
Functions: Fqt-total, Sqw-total, crop max energy: 1MeV
Sassena: \olympic\babylon5\Public\Louise McCann\Simulations\loadSassenaExample.h5
DensityofStates: \olympic\babylon5\Public\Louise McCann\Simulations\squaricn.phonon

Fixes #17025 .



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
